docs/configuration.md

```diff
@@ -968,6 +968,26 @@ still perform some basic text pre-processing before matching.
 
     Defaults to 1.
 
+#### [`PAPERLESS_DATE_PARSER_LANGUAGES=<lang>`](#PAPERLESS_DATE_PARSER_LANGUAGES) {#PAPERLESS_DATE_PARSER_LANGUAGES}
+
+Specifies which language Paperless should use when parsing dates from documents.
+
+    This should be a language code supported by the dateparser library,
+    for example: "en", or a combination such as "en+de".
+    Locales are also supported (e.g., "en-AU").
+    Multiple languages can be combined using "+", for example: "en+de" or "en-AU+de".
+    For valid values, refer to the list of supported languages and locales in the [dateparser documentation](https://dateparser.readthedocs.io/en/latest/supported_locales.html).
+
+    Set this to match the languages in which most of your documents are written.
+    If not set, Paperless will attempt to infer the language(s) from the OCR configuration (`PAPERLESS_OCR_LANGUAGE`).
+
+!!! note
+This format differs from the `PAPERLESS_OCR_LANGUAGE` setting, which uses ISO 639-2 codes (3 letters, e.g., "eng+deu" for Tesseract OCR).
+
+!!! note
+There is a known [issue](https://github.com/scrapinghub/dateparser/issues/875) with `dateparser` related to Chinese locales: many dates are not correctly detected.
+To work around this, if you specify a Chinese locale such as `"zh-Hant"` or `"zh-Hans-HK"`, the generic Chinese language code `"zh"` will also be added automatically.
```

Comment:
```suggestion
```

I totally understand the thought process of including this but tbh I think it's not worth it. If anyone is somehow tripped up by this (I doubt) we can always add a note later. Better to keep the docs short overall though imho.